frontend/src/app/components/containers/complaints/outcomes/hwcr-complaint-assessment.tsx

```diff
@@ -478,17 +479,22 @@ export const HWCRComplaintAssessment: FC<Props> = ({
     if (selectedActionRequired?.value === "No" && !selectedJustification) {
       setJustificationRequiredErrorMessage("Required when Action Required is No");
       return true;
+    } else if (linkedComplaintData?.length > 0 && !linkedComplaintData[0].parent) {
```

Comment:
This is firing as intended for everywhere except the quick close modal from the complaints search list. There it still allows a parent to link as a duplicate and brings its children with it.